docs/pages/enroll-resources/agents/join-services-to-your-cluster/oracle.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Join Services with Oracle Cloud
+description: Use the Oracle join method to add services to your Teleport cluster.
+---
+
+This guide will explain how to use the **Oracle join method** to configure
+Teleport processes to join your Teleport cluster without sharing any secrets
+when they are running in an OCI Compute instance.
+
+The Oracle join method is available to any Teleport process running on an
+OCI Compute instance.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An OCI Compute instance to host a Teleport service.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create the Oracle joining token
+
+Configure your Teleport Auth Service with a special dynamic token which will
+allow services from your OCI tenants to join your Teleport cluster.
+
+Under the hood, services will prove that they are running in your OCI
+tenant by sending a presigned self-authentication request to the OCI API
+for the Auth Service to execute.
+
+Create the following `token.yaml` file with an `oracle.allow` rule specifying
```

Comment:
Let's create a follow-up issue to add a web UI form for creating OCI join tokens.

We don't need to address it right now, but at least this way we have something to track.